src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Machine {
+
+    public static void doWork() {
```

Comment:
The 'doWork' method should have an informative message indicating what type of machine is working. Since 'Machine' is a general class, consider making this method abstract so that each subclass can provide its own informative message.